bigflow/dagbuilder.py

```diff
@@ -66,8 +66,9 @@ def generate_dag_file(
             from airflow.contrib.kubernetes.secret import Secret
             from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
             
-        # BigFlow assumes that you use (airflow 1.x + composer 1.x) or (airflow 2.x + composer 2.x)
+        # BigFlow assumes that we use (airflow 1.x + composer 1.x) or (airflow 2.x + composer 2.x)
```

Comment:
it's just language comment but maybe instead of "BF assumes" you can put it like "To deploy BigFlow project, following requirements options are needed: ...".